client/store/login/actions/localLogin.ts

```diff
@@ -1,22 +1,19 @@
 import { createAsyncThunk } from '@reduxjs/toolkit'
-import { RouteComponentProps } from 'react-router-dom'
 
 import axios from 'axios'
 import { ApiEndPoint } from '@common/api/endpoint'
-import { BasePaths } from '@client/basePaths'
 
 export const localLogin = createAsyncThunk<
   void,
   {
     email: string
     password: string
     invitationUuid?: string
-    history: RouteComponentProps['history']
   }
->('login/local', async ({ email, password, invitationUuid, history }) => {
+>('login/local', async ({ email, password, invitationUuid }) => {
   await axios.post(`${ApiEndPoint.Auth.Login.local()}${invitationUuid ? `?invitationUuid=${invitationUuid}` : ''}`, {
     email,
     password,
   })
-  history.push(BasePaths.Root())
+  window.location.reload()
```

Comment:
why reload?

Reply:
Because after the login a refresh page was necessary to let the system know that the user is logged in, this is just a quick fix. I'm going to create an issue.

Reply:
we have a single page webapp. reload is not necessary.
update page with user information instead

Reply:
done